docs/apps/stable/youtubedl-material/notes.md

```diff
@@ -0,0 +1,12 @@
+# Installation Notes
+
+
+
+## By default Youtube-dl uses the following directories.
+* Audio only files by default downloads to `/app/audio/`
+* Video files by default downlods to `/app/video`
+* Subscriptions by default downloads to `/app/subscriptions/`
+* Multi-user support uses by default `/app/users/`
```

Comment:
needs to be updated and mention those paths are NOT persistent

Reply:
Yea i almost have this ready, but was keeping it for the end, lemme push it and see if there is anything to imporve, cause my english is sometimes ... bad to explain simple things

Reply:
mostly just the fact it's /data/* now, not apps ;-)